py/src/braintrust/framework.py

```diff
@@ -99,9 +114,14 @@ class Evaluator:
     """
 
     """
-    The name of the evaluator. This corresponds to a project name in Braintrust.
+    A unique name for the eval.
```

Comment:
Maybe something like `A name that uniquely defines this type of experiment. You do not need to change it each time the experiment runs, but you should not have other experiments in your code with the same name.`